### ISSUE TITLE
documentation show auto-self links

### ISSUE THREAD

**Issue by generalov** [cainus/percolator#82]
I try example from http://percolatorjs.com/documentation.html#helloworld . Actually the server responds with:

```
{
    "message": "Hello World!",
    "_links": {
        "parent": {
            "href": "http://localhost:3000"
        }
    }
}
```

(note `parent` instead of `self`) https://github.com/cainus/percolator/blob/master/docs/documentation.html#L243

It seems related to #80

ps: Is it possible to prevent the `parent` link generation too?

**Comment by cainus** [comment]
Thanks!  I missed that reference in the docs.  It should be fixed now.

**Comment by generalov** [comment]
Thanks!

**Comment by cainus** [comment]
>  ps: Is it possible to prevent the parent link generation too? 

Put `autoLink : false` in the constructor options.  I just fixed this in 1.4.3.  See http://percolatorjs.com/documentation.html#server for the docs, or let me know if you have any questions.
